env/backend.local.example.env

```diff
@@ -21,7 +21,7 @@ MAILGUN_SENDER_DOMAIN=open.odl.local
 MAILGUN_KEY=fake
 
 # Set this to False if apisix/keycloak are running locally
-DISABLE_APISIX_USER_MIDDLEWARE=True
+DISABLE_APISIX_USER_MIDDLEWARE=False
```

Comment:
you can probably just remove it from example.env since DISABLE_APISIX_USER_MIDDLEWARE=False is the default and also the normal way to run the app now. Also (definitely not in this pr) we can consider a followup to remove the variable entirely, since this var was added for local development and now apisix/keycloak is sufficiently integrated into the app that local development with users that are authenticated without keycloak doesn't really work anymore